src/components/ListingCard/index.tsx

```diff
@@ -15,24 +15,87 @@ export default function ListingCard({
   isSelected?: boolean;
   onClick?: (id: UUID) => void;
 }) {
+  // Helper function to generate an array of strings for CardTag components
+  const generateCardTags = (): string[] => {
+    const tags = [];
+
+    if (caseData.needs_interpreter && caseData.needs_attorney) {
+      tags.push('Interpreter & Attorney');
+    } else if (caseData.needs_interpreter) {
+      tags.push('Interpreter');
+    } else if (caseData.needs_attorney) {
+      tags.push('Attorney');
+    }
+
+    if (caseData.relief_codes && caseData.relief_codes.length > 0) {
+      tags.push(caseData.relief_codes.join(', '));
+    }
+
+    if (caseData.hours_per_month) {
+      tags.push(`${caseData.hours_per_month} hrs/month`);
+    }
+
+    if (caseData.languages && caseData.languages.length > 0) {
+      tags.push(caseData.languages.join(', '));
+    }
+
+    return tags;
+  };
+
+  function formatAdjudicatingAgency(text: string) {
+    // Split the text into words
+    const words = text.split('_');
+
+    if (words.length === 1) {
+      return text;
+    }
+    // Capitalize the first letter of each word
+    const formattedWords = words.map(
+      word => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase(),
+    );
+
+    // Join the words back together
+    const formattedText = formattedWords.join(' ');
+
+    return formattedText;
+  }
```

Comment:
oop I added a similar helper in `src/utils/helpers.ts` (see [multiselect PR](https://github.com/calblueprint/immigration-justice-project/pull/28/files#diff-d9b0865f2e8f841410cd4faf3e1e4a1fb7cb0b545f583f6adaad5841d7eaa58d)) 
do you want to make something similar or do should we switch later?

Reply:
no worries, I'll go ahead and update